app/assets/stylesheets/forms.scss

```diff
@@ -20,3 +20,34 @@
   position: absolute;
   left: -9999em;
 }
+
+.related-external-links {
+  input[type=text] {
+    margin-bottom: 0px;
+  }
+
+  .fields:last-of-type {
+    margin-bottom: 10px;
+  }
+}
+
+.nested-item {
```

Comment:
I'm not sure how much publisher shares styles with other apps in general, but is there a shared admin component we can put these styles into?